chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,65 @@
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    help = u'Creates AttachmentFinalization for the specified Attachment.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(u'-f', u'--file',
+                    dest=u'filename',
+                    help=u'define file path'),
+        make_option(u'--content_type',
+                    help=u'define content type of file',
```

Comment:
Prosim ta do help textov jednotlivych parametrov napis podrobne co robia. Co sa stane ak nie su uvedene a podobne. K parametrom, ktore maju nejaky specialny format daj aj priklad. Odkial to ma uzivatel commandu vediet?

Tiez, parametre nepopisuj, co ma uzivatel robit, ale co tie parametre su. Popis napametrov nie je todolist pre citatela ale popis funcionality. Cize nie "define content type of file". Ale "Content type of file"

Help texty zacitaj velkym pismenom. Ved si pozri ako je to spravene pri defaultnych parametroch, alebo pri ostatnych commandoch.